src/mesh.cpp

```diff
@@ -121,15 +159,15 @@ std::vector<MeshName> BaseName::findAll(const ExecutionContext &context) const
   if (!context.isParallel()) {
     // Check single timestep/meshfiles first
     // Case: a single mesh
-    if (fs::is_regular_file(_bname + ".txt")) {
+    if (fs::is_regular_file(_bname + ".vtk")) {
```

Comment:
preCICE exports parallel VTK files in different VTK formats.
It uses `.vtk` for serial participants and `.vtu` for parallel participants.

Reply:
preCICEv.2.2.0 export `.vtk` for parallel participants. I keep in mind `.vtu` support and parallel handling for next PRs.

Reply:
Parallel as in parallel execution (using mpi) needs to write `vtu` files and a `pvtu` master-file.
The legacy `vtk` format has no support for parallel formats.